src/styles/form.scss

```diff
@@ -0,0 +1,65 @@
+@mixin focus() {
+  outline: none;
+  border-radius: 3px;
+  border: 1px solid $misty-blue-color;
+}
+
+@mixin placeholder() {
+  font-weight: 300;
+  color: $steel-blue-color;
+  text-shadow: none;
+}
+
+.form {
+  display: flex;
+  padding-bottom: 20px;
+
+  &__input {
+    box-sizing: border-box;
+    width: 100%;
+
+    color: $steel-blue-color;
+    font-size: 16px;
+    font-weight: 800;
+    text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+
```

Comment:
You should add `font-family`
<img width="680" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/49764881/476ccbbf-422a-49ba-9add-db14185264ac">